.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--  Thanks for contributing to Cilium!
+
+If this is your first time contributing, then please see the following
+guidelines for detailed instructions on how to contribute:
+https://github.com/cilium/cilium/blob/master/CONTRIBUTING.md for detailed instructions 
+
+-->
+
```

Comment:
will the first lines show up when opening the PR? Looks a HTML comment.

Reply:
I found this in other repositories that use this. We'll know soon.

Reply:
tried it now. It shows up.